ajax.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="ajax.js"></script>
+</head>
+<center>
+<body style="background-color: lavenderblush;" style="align-items: center;">
+    <h2 style="color: crimson;">HOW TO REQUEST DATA FROM A SERVER</h2>
+    <button type="button" style="color: blue;" onclick="myfunction()">GetData</button>
+    
+<Div id="info"></Div>
+
+</body>
+</center>
+</html>
```

Comment:
Kindly add a new line at the end of all your files as standard practice dictates.